tests/test-dirs/locate/no-wrap.t/run.t

```diff
@@ -0,0 +1,66 @@
+  $ mkdir .cmi-dir
+  $ mkdir .cmi-dir/lib
+
+  $ cd src/lib
+
+  $ $OCAMLC -pack -o libpack.cmo -bin-annot config.ml lib.ml
+
+  $ ls
+  config.cmi
+  config.cmo
+  config.cmt
+  config.ml
+  lib.cmi
+  lib.cmo
+  lib.cmt
+  lib.ml
+  libpack.cmi
+  libpack.cmo
+  libpack.cmt
+
+  $ cp libpack.cmi ../../.cmi-dir/lib
+
+  $ cd ../bin
+
+  $ $OCAMLC -o exe.exe -bin-annot -I ../../.cmi-dir/lib \
+  > ../lib/libpack.cmo config.ml exe.ml
+
+  $ ./exe.exe
+  Libpack.Lib.config_value: 4242,
+  Libpack.Config.value: 4242,
+  Config.value: 42
+
+  $ cat >.merlin <<EOF
+  > CMI ../../.cmi-dir/lib
+  > CMI .
+  > CMT .
+  > CMT ../lib
+  > EOF
+
+FIXME should jump to src/lib/lib.ml
+  $ $MERLIN single locate -look-for ml -position 2:20 \
+  > -filename ./exe.ml <exe.ml | jq '.value'
+  "'Libpack.Lib.config_value' seems to originate from 'Lib' whose ML file could not be found"
+
+
+FIXME should jump to src/lib/config.ml
```

Comment:
I think it is as expected, the shapes are reduced to some `<Config.0>` but since we do not load the load_path of the pack it jumps to the wrong config:

```
   $ $MERLIN single locate -look-for ml -position 3:20 \
   > -log-file - -log-section locate -filename ./exe.ml <<EOF
   >  Printf.eprintf "Libpack.Lib.config_value: %i,\nLibpack.Config.value: %i,\nConfig.value: %i\n"
   > Libpack.Lib.config_value
   > Libpack.Config.value
   > Config.value
   > EOF
   
+  # 0.01 locate - reconstructed identifier
+  Libpack.Config.value
+  # 0.01 locate - from_string
+  inferred context: expression
+  # 0.01 locate - from_string
+  looking for the source of 'Libpack.Config.value' (prioritizing .ml files)
+  # 0.01 locate - lookup
+  lookup in value namespace
+  # 0.01 locate - env_lookup
+  found: 'Libpack!.Config.value' in namespace value with uid Config.0
+  # 0.01 locate - shape_of_path
+  initial: CU Libpack . "Config"[module] . "value"[value]
+  # 0.01 locate - read_unit_shape
+  inspecting Libpack
+  # 0.01 locate - find_in_path_uncap
+  Failed to load $TESTCASE_ROOT/src/bin/Libpack.cmt
+  # 0.01 locate - read_unit_shape
+  shapes loaded for Libpack
+  # 0.01 locate - read_unit_shape
+  inspecting Config
+  # 0.02 locate - read_unit_shape
+  shapes loaded for Config
+  # 0.02 locate - shape_of_path
+  reduced: <Config.0>
+  # 0.02 locate - from_uid
+  Loading the shapes for unit "Config"
+  # 0.02 locate - File_switching.move_to
+  $TESTCASE_ROOT/src/bin/config.cmt
+  # 0.02 locate - from_uid
+  Shapes succesfully loaded, looking for Config.0
+  # 0.02 locate - from_uid
+  Found location: File "config.ml", line 1, characters 4-9
+  # 0.02 locate - find_source
+  attempt to find "config.ml"
+  # 0.02 locate - result
+  found: $TESTCASE_ROOT/src/bin/config.ml
```

Reply:
(also your comment was not made on the final version of the test which has one of the two failures fixed)